### DISCUSSION TITLE
How to add zoom level to FlyTo button?

### DISCUSSION THREAD

**Discussion by alizezila** [albertkun/21S-ASIAAM-191A#124] [Help Desk] [answered]
Hey all! Does anyone know how to coordinate a zoom level for when I click on a button and it takes me to my point marker? I'd like to click the button and it flies to my location as well as zooming in closer than my set zoom level. Sorry if post is confusing. Thanks!

**Comment by albertkun** [comment, marked as answer]
@alizezila in the `FlyTo()` you can specify the Zoom level as the second parameter, basically something like:
`FlyTo([33,-112],10)` would zoom into Zoom level 10 at 33 Latitude/-112 Longitude.

You can see the parameters here:

https://leafletjs.com/reference-1.6.0.html#map-flyto

**Reply by alizezila** [reply]
works perfectly! Thanks so much :)

**Reply by albertkun** [reply]
@alizezila you are very welcome!
